main_window/packet_parser.py

```diff
@@ -40,21 +43,24 @@
 PACKET_ID_TO_CONST_LENGTH: Dict[int, int] = {
     SubpacketEnum.STATUS_PING.value: 5,
     SubpacketEnum.EVENT.value: 1,
+    SubpacketEnum.CONFIG.value: 3,
     SubpacketEnum.GPS.value: 24,
-    # SubpacketEnum.ACKNOWLEDGEMENT.value: 0000, # TODO
+    # SubpacketEnum.ACKNOWLEDGEMENT.value: 0000, # TODO Waiting on spec
     SubpacketEnum.BULK_SENSOR.value: 37,
 }
```

Comment:
You're right! Sorry, had my head down in the code

Reply:
Wait, are we not getting rid of the PACKET_ID_TO_CONST_LENGTH dict? Do we still need it for anything?

Reply:
We can get rid of the use, but it exists more as a reference for the parser to know if there will be a variable length or not. It could become a bool dict essentially, but we will need something to know which packets are variable

Reply:
Ah right, I see. On the FW the length is treated as less of the last optional property in the header and more like the first property of certain packet types. So it might be cleaner to just parse the length for the packets that use it that packet's parser function. Especially since I think only 1 or 2 packets have variable length.